lib/cli.js

```diff
@@ -70,7 +70,7 @@ const main = async () => {
     if (cliOptions.hook) {
       const commitMsgFile = join(getGitRootDir(), '.git', 'COMMIT_EDITMSG');
 
-      fs.writeFileSync(commitMsgFile, message);
+      fs.writeFileSync(commitMsgFile, message.join('\n\n'));
```

Comment:
I'm thinking maybe in `--dry-run` block we should also display how the message should look like.

```js
    if (cliOptions.dryRun) {
      const command = shellescape(['git', ...executeCommandArgs]);

      /* eslint-disable no-console */
      console.log('Will execute command:');
      console.log(command);
      console.log('Message:');
      console.log(message.join('\n\n'));
      /* eslint-enable no-console */
    }
```

Reply:
Sure, sounds good. I can throw that in in a little bit